app/templates/board.html

```diff
@@ -0,0 +1,24 @@
+<!-- Author: Micah Weiberg
+     Version: 05-15-19
+     HTML file with the Jinja for loops to display the 8x8 grid of the chessboard
+     and allow color changes depending on data sent from views/__init__.py -->
+<table class="chessboard">
+    {% for row in range(8) %}
+        {% for col in range(8) %}
+            {% if row is even() %}
+                {% if 'true' in data %}
+                    <th class="{{ loop.cycle('black', 'white') }}"></th>
+                {% else %}
+                    <th class="{{ loop.cycle('white', 'black') }}"></th>
```

Comment:
You probably could write this with even fewer lines, due to using loop.cycle. All you need is the starting color.